scripts/startNewSchoolYear.js

```diff
@@ -41,6 +41,7 @@ appPromise
 				{
 					federalState: { $in: federalStateIds },
 					ldapSchoolIdentifier: { $exists: false },
+					currentYear: { $exists: true },
```

Comment:
I wonder if we should make it even more strict, so to only do the update, if the currentYear is really the current year we would define on top. So, then you see the update from e.g. 21/22 -> 22/23

Reply:
agree in general, but in this case it would also apply the update to schools being (for whatever reason) stuck in old school years (which was the case already) and so it would just take all of them as it makes no sense (out of the thombstone school) to stay in an old school year